### ISSUE TITLE
Failed to Set Up: Import Error

### ISSUE THREAD

**Issue by aiden21c** [mvdwetering/yamaha_ynca#298]
**Describe the bug**
Recently updated from I believe a version prior to 7.0 and I now get an import error on the integration preventing me from doing anything at all. The entire integration no longer functions.

**To Reproduce**
1. Update to version 7.9.0
2. Integration no longer works

**Logging**
Main issue appears to be here 
[home-assistant_yamaha_ynca_2024-07-14T06-09-51.764Z.log](https://github.com/user-attachments/files/16215851/home-assistant_yamaha_ynca_2024-07-14T06-09-51.764Z.log):
```
AttributeError: type object 'Input' has no attribute 'AUDIO'. Did you mean: 'AUDIO1'?
2024-07-14 16:09:47.190 ERROR (MainThread) [homeassistant.config_entries] Error importing platform config_flow from integration yamaha_ynca to set up yamaha_ynca configuration entry: Exception importing custom_components.yamaha_ynca.config_flow
```

- Home Assistant Core version: core-2024.7.2
- Home Assistant operating environment (OS/Container/Supervised/Core): OS
- Integration version: 7.9.0
- Last working version (if known): Prior to 7.0
- Receiver: RX-V773

**Comment by mvdwetering** [comment]
It looks like the library that the integration is using for communicating with the receiver is old. The AUDIO input was added in February 2024, so should be available if the correct version of the library is installed.

Home Assistant is supposed to check and install the correct version when it starts.
I don't see errors about downloading the library, so not sure why it is not updating the library.

Did you already try restarting Home Assistant?

**Comment by aiden21c** [comment]
Just confirming I have restarted HA using the UI. I am running HA OS in a VM if that makes a difference.

**Comment by aiden21c** [comment]
I attempted to remove and re-add the integration from HACS and I now get this error when adding it from the integrations page:
```
Error
Config flow could not be loaded: {"message":"Invalid handler specified"} 
```

**Comment by mvdwetering** [comment]
It kind of seems like a corrupt download, but I think HACS checks the downloaded file.
Also I just redownloaded reinstalled 7.9.0 and I can still add a new receiver, so download itself seems fine.

I don't understand what is going on. 

Is there anything else in the logs?
What happens if you try a slightly older version like 7.8.0?

**Comment by aiden21c** [comment]
[home-assistant_2024-07-21T12-15-17.765Z.log](https://github.com/user-attachments/files/16324384/home-assistant_2024-07-21T12-15-17.765Z.log)

I've now tried everything I could think of. I've tried every version from 7.6.0 to 7.9.0, including removing and redownloading the entire integration in between. I've also tried this while rebooting the entire VM between each integration install. Nothing I have done seems to have been able to get the integration up and running, and I am hitting the same issue. I've attached the logs for reference.

I really have no clue whats going on either.

**Comment by mvdwetering** [comment]
With the logging you provided it seems like Home Assistant does not even load the integration properly. Therefore I made a small debug integration that will just print the version and location of the ynca package since there seems to be an inconsistency.

First make sure the `yamaha_ynca` integration is at 7.9.0 even if it is not working properly.

Install the attached ynca_debug integration by extracting it into the custom_components directory. 
Then in `configuration.yaml` add the line `ynca_debug:` so it will be loaded.

Next restart Home Assistant

When you look in the logs there should be a line with a warning like below. Please copy/paste that line here. 

```
***** ynca package info, version 5.15.0, location /usr/local/lib/python3.12/site-packages/ynca/__init__.py
``` 

Afterwards you can delete the line in configuration.yaml and the ynca_debug integration and restart Home Assistant. To clean up.

[ynca_debug.zip](https://github.com/user-attachments/files/16338517/ynca_debug.zip)

**Comment by aiden21c** [comment]
Thank you so much for your help. Here is the line you requested:
```
 ***** ynca package info, version 5.9.0, location /config/deps/lib/python3.12/site-packages/ynca/__init__.py 
```

I have also attached the full logs 
[home-assistant_2024-07-23T02-57-40.167Z.log](https://github.com/user-attachments/files/16343176/home-assistant_2024-07-23T02-57-40.167Z.log)

**Comment by mvdwetering** [comment]
So Home Assistant indeed loads an old version of the ynca package. 

The location `/config/deps/...` is intended to (temporarily) override packages for development purposes. I don't know why/how  you got a ynca package there. 

I would suggest to delete at the whole `/config/deps/lib/python3.12/site-packages/ynca`  directory.  Then Home Assistant will load it from the standard location and everything should be fine.

If  you have more packages in the `config/deps/...` directory you might want to consider deleting it to avoid future problems (keep a backup if you are not sure why it is there).

**Comment by mvdwetering** [comment]
FYI I just released v7.10.0 of this integration. 

It does not fix this issue, but just wanted to mention that you should remove the `ynca_debug` integration if not already done as it will conflict with the new release.

**Comment by aiden21c** [comment]
Thank  you so much for your help. Ive finally got around to trying your suggestion and it seems to have sorted everything out. I think maybe when i moved from HA in docker to a dedicated HAOS some things might have gotten jumbled up with HACS integrations. Thank you again for all your assistance.
